network/tc_test.go

```diff
@@ -87,7 +87,7 @@ var _ = Describe("Tc", func() {
 
 		Context("add 1s delay and 1s delayJitter to lo interface to the root parent without any handle", func() {
```

Comment:
Would it be possible to add a test that confirms the default 10% is set appropriately? especially at or near the lower bounds of `delay`? Otherwise I'm +1 on the PR

Reply:
After looking at the code a bit, I noticed that adding the test would require more changes from the original code. The 10% computation is not done within tc.go, specifically the AddNetem function. It is done within injector/network_disruption.go. Therefore in tc_test.go we cannot easily test it because those tests dont go through the Inject code in injector/network_disruption.go. What are your thoughts?